REQUIRE

```diff
@@ -1 +1 @@
-julia 0.6
+julia 0.6 0.7.0-DEV.602 0.7.0-DEV.3449
```

Comment:
This syntax is wrong; there should only be two versions listed. (How did this not cause an error?)

Reply:
It's correct (but admittedly rarely used), see https://docs.julialang.org/en/latest/manual/packages/#Requirements-Specification-1. It means that this package may be installed if `v"0.6" <= VERSION < v"0.7.0-DEV.602" || v"0.7.0-DEV.3449" <= VERSION`.

Reply:
Whoa, holy cow. TIL.